src/app/system/system.module.ts

```diff
@@ -114,7 +115,18 @@ import { ConfigureMakerCheckerTasksComponent } from './configure-maker-checker-t
     EditRoleComponent,
     EntityToEntityMappingComponent,
     ConfigureMakerCheckerTasksComponent,
-    CreateSurveyComponent
-  ]
+    CreateSurveyComponent,
+    EditSchedulerJobComponent,
+    ViewHistorySchedulerJobComponent
+  ],
+  entryComponents: [
+    ReportParameterDialogComponent,
+    AddEventDialogComponent,
+    ColumnDialogComponent,
+    EditHookComponent,
+    ViewHookComponent,
+    CreateHookComponent,
+    AddEventDialogComponent
+  ],
```

Comment:
```suggestion
 
```
Entry components declaration is no longer required.

Reply:
I'll remove them all